### DISCUSSION TITLE
Is there any way to make dark mode the default?

### DISCUSSION THREAD

**Discussion by Andrew-S-Rosen** [pradyunsg/furo#671] [Questions] [answered]
If I am using furo, is there a way that I can make the default mode be the dark style (while still allowing the user to toggle to light mode)?

**Comment by pradyunsg** [comment, marked as answer]
Use dark mode on the operating system?

https://pradyunsg.me/furo/customisation/colors/#how-light-and-dark-mode-work and https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-color-scheme

**Reply by Andrew-S-Rosen** [reply]
That only influences me though. If I'm deploying the docs and want dark mode to be the default for all users, that won't work then, right?

**Reply by Eric-Arellano** [reply]
You can use code like this:

https://github.com/Qiskit/qiskit_sphinx_theme/blob/57c4bee5bd85fadcdb4e4364922d1c00aa698b49/qiskit_sphinx_theme/furo/base/custom_templates/extra_head.html#L5-L10

But I agree with what Pradyun has said in the past. It's an antipattern to not respect the user's preferences for dark mode vs light mode. We, unfortunately, had to temporarily disable dark mode in our project because it's a much bigger project to add dark mode to our entire site (docs are only a sub-part) -- but it's not ideal.

**Reply by pradyunsg** [reply]
I understand that documentation authors will have stylistic preferences -- I also wanna be mindful of, and respect, the readers' preferences which they do configure their OS & browser to handle this stuff. This is primarily opinionated -- I think giving the readers more control over the reading experience is better than giving documentation authors more control.

It is possible to change up how stuff works, but that's tedious _because_ I don't think it's possible to cater to all the preferences arbitrary groups of people might have for that. I do understand that folks _want_ to be able to do things like that, so I don't intend to be disruptive around that stuff for no reason. :)
